lib/views/player/widgets/bottom_actions/bottom_action_widget.dart

```diff
@@ -22,11 +23,12 @@ class BottomActionWidget extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.center,
         mainAxisAlignment: MainAxisAlignment.spaceAround,
         children: [
-          AudioSpeedWidget(),
-          width8,
-          AudioDownloadWidget(
-            trackModel: trackModel,
-            file: file,
+          SizedBox(
+            width: 60,
```

Comment:
Why 60? I thought they were icons evenly spaced out

Reply:
The icons were moving left and right due to variable text width. So, I thought providing a fixed width will solve this issue.

Reply:
Hardcoded width removed thanks for the help man